python_modules/dagster/dagster/_core/definitions/asset_daemon_cursor.py

```diff
@@ -221,6 +235,15 @@ def from_serialized(cls, cursor: str, asset_graph: AssetGraph) -> "AssetDaemonCu
             except:
                 subset = partitions_def.empty_subset()
             handled_root_partitions_by_asset_key[key] = subset
+
+        latest_evaluation_by_asset_key = {}
+        for key_str, serialized_evaluation in serialized_latest_evaluation_by_asset_key.items():
+            key = AssetKey.from_user_string(key_str)
+            evaluation = deserialize_value(serialized_evaluation)
+            if not isinstance(evaluation, AutoMaterializeAssetEvaluation):
```

Comment:
When would this happen? It it shouldn't happen, we should probably error instead?

Reply:
hm yeah sounds reasonable